api/numaresourcesoperator/v1/helper/nodegroup/nodegroup_test.go

```diff
@@ -420,3 +461,20 @@ func findListByNodeGroups(mcps *mcov1.MachineConfigPoolList, nodeGroups []nropv1
 
 	return result, nil
 }
+
+// containedItemsFromSlice returns true if all of b items are part of slice a
+func containsSliceItems(a, b []string) bool {
```

Comment:
we can use the `slices` package  I think: https://pkg.go.dev/slices@go1.22.0
(and yes, in the future we should scan our codebase and move possibly all of the places to `slices`)